### ISSUE TITLE
Naming conventions?

### ISSUE THREAD

**Issue by kfrasier** [MarineBioAcousticsRC/Triton#6]
@asolsonaberga Do you think we should figure out a naming convention for all of the remoras we have? 
I think it could get confusing if things end up with the same names, etc. Maybe we could discuss briefly.

**Comment by asolsonaberga** [comment]
I agree! I tried to put similar names but that each remora have the distinctive word, for e.g. dt_ship_control for Ship-Detector, whereas for the others is dt_control for Spice-Detector, and dtcontrol for Detector Remora. So I think we should at least add the main word of the remora in any script.

But on the other hand, when you add all the remoras, the code of the last remora added has the highest priority, which can be advantageous. For example, for Ship Detector I had to add the specific plot_ltsa that will overwrite the default in Triton, that way I can plot the tlab files.

**Comment by asolsonaberga** [comment]
@kfrasier! dt_.... It is more accurate for the DetectorRemora. So maybe files could be named like

ImageSet Remora -  prefix: is -> is_control
Logger Remora -  prefix: log -> log_control
Spice Detector Remora -  prefix: sp -> sp_control
Ship Detector Remora -  prefix: sh -> sh_control
Cluster Remora -  prefix: ct -> ct_control
Detector Remora -  prefix: dt -> dt_control

**Comment by kfrasier** [comment]
Perfect. I'll start applying that to spice, imageset, cluster, and dt over
the next few weeks. Can do logger too eventually.
For upcoming remoras, I'll run a proposed prefix by you before initial
commit.

**Comment by asolsonaberga** [comment]
I can do dt because I am working on it and I have already a branch with new code

**Comment by asolsonaberga** [comment]
The only thing I do not know how to do is when overwriting the default scripts, lie the plot_ltsa. It needs to be the same name, and when you add the remora, you need to close all the figure of triton and restart to make sure that the new script is on top of the old one. I can't find a better way to do that

**Comment by kfrasier** [comment]
That case seems ok for right now. Probably we will think of a better
solution eventually. Maybe it would be good though to at least print a
message notifying the user and warning about unintended consequences? I'm
thinking about R where this happens a lot - one function from a package
blocks another function with the same name. It's annoying, but I guess it
works ok most of the time.
